tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -1053,3 +1053,26 @@ def tearDown(self):
         self.apps = None
         self.data_layer = None
         MarionetteTestCase.tearDown(self)
+
+
+class PasscodeTestCase(GaiaTestCase):
+    def setPasscodeTo1337(self):
+        " this  just sets the passcode. does not enable or disable it. "
```

Comment:
We should make this a valid docstring, see http://legacy.python.org/dev/peps/pep-0257/#one-line-docstrings